### ISSUE TITLE
New lines

### ISSUE THREAD

**Issue by vdavez** [konklone/json#10]
I think new lines (pretty JSON) in the JSON break the conversion.

**Comment by konklone** [comment]
Show me the JSON?

Also, if you're copying straight from JSONView, there's added formatting that breaks it. I "View Source" from JSONView before copying.

**Comment by konklone** [comment]
Without a concrete example, going to close the issue.

**Comment by vdavez** [comment]
BTW--You were right. It was copying from JSONView that got me into trouble...

**Comment by konklone** [comment]
In the last few days, I've learned that [JSON Formatter](https://chrome.google.com/webstore/detail/json-formatter/bcjindcccaagfpapjjmafapmmgkkhgoa?hl=en) is a better extension for Chrome, which doesn't have the copy/paste problem. JSONView for Firefox is also immune to the copy/paste problem (the Firefox and Chrome extensions are different codebases by different people). 

Pasting from JSONView for Chrome will actually work in many cases now anyway, as I added an `eval()` fallback if parsing fails on the first pass. This allows valid JS objects which aren't valid JSON objects (basically, unquoted keys) to now pass inspection. JSONView for Chrome's formatted display does not properly escape quotes inside strings though, which is not a work-around-able problem if the JSON in question happens to contain them.
